plugins/fabric8-cdn-support/src/main/java/com/redhat/che/cdn/CdnSupportService.java

```diff
@@ -146,27 +132,40 @@ public String getPaths() throws Exception {
       throw new NotFoundException("No editor is configured for CDN resource pre-fetching");
     }
 
-    String url = retrieveEditorPluginUrl();
-    if (!url.equals(editorDefinitionUrl)) {
-      editorDefinitionUrl = url;
-      dockerImage = null;
-    } else {
-      LOG.debug("Editor full definition didn't change");
+    PluginMeta editorMeta = getEditorMeta();
```

Comment:
Thinking from a performance perspective, is it possible to add some form of caching here? Doing this whole process constantly when we expect the relevant data to change rarely seems like a waste.

This would probably be a separate issue, though.

Reply:
good point. Lemme update a bit

Reply:
@amisevsk PR updated